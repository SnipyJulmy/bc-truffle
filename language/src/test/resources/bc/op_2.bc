1 + \
23
32 \
+ \
2
2\
2 + 3217
halt
