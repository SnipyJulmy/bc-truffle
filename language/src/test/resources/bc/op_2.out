24
34
3239
